modules/fts/pages/fts-highlighting.adoc

```diff
@@ -0,0 +1,10 @@
+= Highlighting
+
+The `Highlight` object indicates whether highlighting was requested. The highlight object contains the following fields:
```

Comment:
Add a line here, for the pre-requisite -

`include term vectors` and `store` options need to be enabled at the field level to support highlighting.